components/landing/TheLearnSection/LearnCard.vue

```diff
@@ -33,39 +33,51 @@ export default class LearnCard extends Vue {
 
 <style lang="scss" scoped>
 .learn-card {
-  height: 16rem;
-  width: 100%;
-  background-color: $background-color-lighter;
   display: flex;
+  flex-wrap: wrap;
 
-  @include mq($until: large) {
-    height: auto;
+  @include mq($from: medium) {
+    height: $spacing-10 * 3;
```

Comment:
The `spacing` values are meant to be [used for margin and padding](https://carbondesignsystem.com/guidelines/spacing/overview/#applying-spacing). I think `height` should have the value directly on rems